layersvt/api_dump.h

```diff
@@ -241,9 +242,69 @@ class ConditionalFrameOutput {
     }
 };
 
+#ifdef __ANDROID__
+
+class AndroidLogcatBuf : public std::streambuf {
```

Comment:
We can make this class more generic, so that it can accept `wchar_t`, `char8_t`, `char16_t`, `char32_t`.